documentation/docs/20-core-concepts/50-state-management.md

```diff
@@ -113,6 +113,8 @@ You might wonder how we're able to use `$page.data` and other [app stores](modul
 <p>Welcome {$user.name}</p>
 ```
 
+Updating the context-based store value in deeper-level pages or components will not affect the value in the parent component when the page is rendered via SSR. To avoid values 'flashing' during state updates during hydration, it is generally recommended to pass state down into components rather than up.
```

Comment:
```suggestion
Updating the context-based store value in deeper-level pages or components will not affect the value in the parent component when the page is rendered via SSR: The parent component has already been rendered by the time the store value is updated. To avoid values 'flashing' during state updates during hydration, it is generally recommended to pass state down into components rather than up.
```